float64 longtitude
float64 latitude
float64 gps_time
float64 altitude

